topics/admin/tutorials/pulsar/tutorial.md

```diff
@@ -72,19 +73,152 @@ In this tutorial, we will:
 This tutorial assumes that you have:
 
 - A VM or machine where you will install Pulsar, and a directory in which the installation will be done. This tutorial assumes it is `/mnt`
-- That you have completed the "Galaxy Installation with Ansible" and CVMFS tutorials (Job configuration tutorial is optional.)
+- That you have completed the "Galaxy Installation with Ansible" and CVMFS tutorials (Job configuration tutorial is optional) and have access to the VM/computer where it is installed.
+
+# Overview
+
+We will be installing the RabbitMQ server daemon onto the Galaxy server to act as an intermediary message passing system between Galaxy and the remote Pulsar. The figure below shows a schematic representation of the system.
+
+![pulsar_australia.png](../../images/pulsar_amqp_schema.png)
+
+**Figure 1: Schematic diagram of Galaxy communicating with a Pulsar server via the RabbitMQ server.**
+
+**How it will work:** 
+
+1. Galaxy will send a message to the RabbitMQ server on the Pulsar server's particular queue saying that there is a job to be run and then will monitor the queue for job status updates.
+2. The Pulsar server monitors this queue and when the job appears it will take control of it.
+3. The Pulsar server will then download the required data etc. from the Galaxy server using `curl`.
+4. The Pulsar server will install any required tools/tool dependencies using Conda.
+5. The Pulsar server will start running the job using it's local mechanism and will send a message to the "queue" stating that the job has started.
+6. Once the job has finished running, the Pulsar server will send a message to the queue stating that the job has finished.
+7. Pulsar then sends the output data etc. back to the Galaxy server by `curl` again.
+8. The Galaxy server acknowledges the job status and closes the job.
+
+**Some notes:**
+
+* RabbitMQ uses the Advanced Message Queueing Protocol (AMQP) to communicate with both the Galaxy server and the remote Pulsar VM.
+* Transport of files, meta-data etc. occur via `curl` from the Pulsar end.
```

Comment:
We'll get questions here, we should head them off by including a tip box with other methods to share files.